ProphetBot/models/db_objects.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, **kwargs):
             setattr(self, key, value)
 
     def get_name(self, ctx: ApplicationContext):
-        return discord.utils.get(ctx.bot.get_all_members(), id=self.player_id).name
+        return discord.utils.get(ctx.bot.get_all_members(), id=self.player_id).mention
```

Comment:
Not necessarily a problem with this change specifically, but this runs the risk of blowing up if the member isn't found for whatever reason (most likely leaving the server). The .get() would return `None`, which would throw an `AttributeError` when you try to access the mention attribute.

It's a bit niche, but I'd recommend handling that to return None (or something more descriptive/helpful) instead of the exception you would currently get.

Reply:
Added in a try/catch for now. probably could do this better, but will worry about that when re-vamping everything because I think the adventure embeds have the same issue.

Reply:
Adventure stuff is old and probably messy, yeah. Looks good enough for now.